src/encode/encode_float.inl.h

```diff
@@ -967,465 +968,4 @@ force_inline u8 *write_u64(u64 val, u8 *buf) {
     }
 }
 
-/*==============================================================================
```

Comment:
Please check if we can remove this file entirely? If not, move needed functions to other header file, e.g. `encode_shared.h`